### ISSUE TITLE
June 21. Meeting ToDos

### ISSUE THREAD

**Issue by sudo-Boris** [timherzig/decomposition_learning#15]
## Clean up code base
* Merge all branches into one (then into main)
* Extract functions and make modular

## Training debugging
* Sanity check for loss functions -> overfit on one training image

## Architecture changes
* (Optional) Try a smaller model without downscaling -> compression loses too much information
* Change regularized loss
     * log scale
     * remove/ make weaker: mask decay
     * "completely" rethink extreme cases and tune loss regularizations.
     * SSIM loss??? (for occlusion?)
* Change Unet branches final activations (see last meeting notes)
